### DISCUSSION TITLE
[Feature] Allow unhiding a face that is hidden in the current picture

### DISCUSSION THREAD

**Discussion by chodthewacko** [immich-app/immich#8548] [Feature Request]
### The feature

EDIT:   why can't you delete requests?

Turns out, you can do this:
in the picture, go to:
info -> eye (i.e. show hidden people), then you can click on the hidden person to go to their 'face profile'
then the top right 'three dots' menu, then 'unhide person'

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by DanTulovsky** [comment]
I don't seem to be able to do this... can you show a screenshot?  I just have this and no way to unhide:

<img width="213" alt="image" src="https://github.com/user-attachments/assets/d7d7b007-a7b9-44d6-9cec-14428366604b">

**Comment by Micha10** [comment]
I have that problem with some pictures. There are persons which are not recognized to be the same person so I want to join them. I open a photo. The icon states that there are hidden persons, After clicking "edit" the same picture as above is shown. I can't click on the person to navigate to their page. I just can "-" minus them and create a new person but not name or join them.
